javascript/javascript1/week4/js1-week4-homework/index.js

```diff
@@ -0,0 +1,92 @@
+const name = [];
+const toDo = [];
+function getReply(command) {
+  const lowerCase = command.toLowerCase();
+  //console.log(lowerCase)
+
+  let words = lowerCase.split(" ");
+  let myName = words[words.length - 1];
+  name.push(myName);
+  //console.log(name)
+  if (lowerCase.includes("hello my name is")) {
```

Comment:
You could also consider [startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith)